internal/bminventory/inventory.go

```diff
@@ -354,6 +354,22 @@ func (b *bareMetalInventory) validateRegisterClusterInternalParams(params *insta
 	return nil
 }
 
+func (b *bareMetalInventory) validateAccessToARMFunctionality(ctx context.Context, cpuArchitecture string) error {
+	var err error
+	var armArchAllowed bool
+
+	if cpuArchitecture == "arm64" {
```

Comment:
could be a switch/case instead in case we add other archs later

Reply:
Done.